### PR TITLE
OCPBUG13868: Added a note related to OVS HWOL.

### PR DESCRIPTION
**Pull request by subhtk**
Version(s): 4.11+

Issue: [OCPBUG 13868](https://issues.redhat.com/browse/OCPBUGS-13868)

Link to docs preview: [Preview](https://62334--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/network-configuration.html#nw-osp-enabling-ovs-offload_post-install-network-configuration)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by HareshKhandelwal — COMMENTED** [review]
True, ALG flows wont be offloaded.

**Comment by abhatt-rh** [comment]
Hi @subhtk Can you please upload the preview link for this PR?

**Review by abhatt-rh — COMMENTED** [review]
Left a comment for your consideration.

File: modules/nw-osp-enabling-ovs-offload.adoc [hunk 1]
```diff
@@ -119,3 +119,8 @@ spec:
   - name: dpdk-testpmd
     image: quay.io/krister/centos8_nfv-container-dpdk-testpmd:latest
 ----
+
+[NOTE]
+====
+Application-layer gateway flows are broken in OpenShift Container Platform version 4.10, 4.11, and 4.12 and are not offloadable for OpenShift Container Platform version 4.13.
```
[the remark this answers is not in the document]
Comment (subhtk) [reply modules/nw-osp-enabling-ovs-offload.adoc:125]: Ack. Let me change it right away.

**Review by jeana-redhat — APPROVED** [review]
LGTM

/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Pull request merged**